content/docs/client-options.md

```diff
@@ -29,6 +29,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [GetSSL](https://github.com/srvrco/getssl/tree/APIv2) (`APIv2` branch)
 - [acme.sh](https://github.com/Neilpang/acme.sh/tree/2) (`2` branch)
 - [Net::ACME2](https://metacpan.org/pod/Net::ACME2)
+- [EasyHTTPs](https://easy.zhetao.com) (Automatically select v2 or v1)
```

Comment:
It looks like EasyHTTPS charges money for all of the available service levels. That's certainly OK and within your rights, but we've decided to start specifically noting the clients that cost money on the `client-options.md` page to help make users aware of the cost up-front.

Could you please add an `*` at the end of each of the `[EasyHTTPs]` links and a small note at the bottom of the page like "* = Service may require payment" or something similar?

Thank you!

Reply:
OK, I will add the * tag and resubmit the PR.

Thank you!